CDPHW1/src/com/epam/cdp/java_testng/iryna_ramanets/task1/IntArrayHandler.java

```diff
@@ -0,0 +1,36 @@
+/**
+ * 
+ */
+package com.epam.cdp.java_testng.iryna_ramanets.task1;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Class to perform operations with arrays of integers
+ */
+public class IntArrayHandler {
+	
+	private List<Integer> arr;
+
+	public IntArrayHandler(List<Integer> arr) {
+		this.arr = arr;
+	}
+
+	/**
+	 * Find elements that occur more than once and have odd index
+	 * @param arr input array
+	 * @return list of element or null if input null
+	 */
+	public List<Integer> findMultiElementsWithOddIndex() {
+		if(arr == null) return null;
```

Comment:
you shouldn't return null. Return empty list.
so also If you use Java 8 and Collection framework please use new features like that:
```
public List<Integer> findMultiElementsWithOddIndex() {

		return arr.stream().filter(element -> {
            int index = arr.indexOf(element);
            return (index != arr.lastIndexOf(element)) && (index % 2 == 1);
		}).collect(Collectors.toList());
	}
```

Feel free to ask me about what it means. But we will touch all the things again when will work with Collection framework in Java